packages/block-editor/src/components/block-draggable/index.native.js

```diff
@@ -281,50 +239,26 @@ const BlockDraggableWrapper = ( { children } ) => {
  * @return {Function} Render function which includes the parameter `isDraggable` to determine if the block can be dragged.
  */
 const BlockDraggable = ( { clientId, children } ) => {
-	const { blocksLayouts, findBlockLayoutByClientId } = useBlockListContext();
+	const { selectBlock } = useDispatch( blockEditorStore );
+	const wasBeingDragged = useRef( false );
 
-	const collapseAnimation = {
-		opacity: useSharedValue( 0 ),
-		height: useSharedValue( 0 ),
-		initialHeight: useSharedValue( 0 ),
+	const draggingAnimation = {
+		opacity: useSharedValue( 1 ),
 	};
 
-	const startBlockDragging = () => {
-		const blockLayout = findBlockLayoutByClientId(
-			blocksLayouts.current,
-			clientId
+	const startDraggingBlock = () => {
+		draggingAnimation.opacity.value = withTiming(
+			0.4,
+			BLOCK_OPACITY_ANIMATION_CONFIG
 		);
-		if ( blockLayout?.height > 0 ) {
-			collapseAnimation.initialHeight.value = blockLayout.height;
-			collapseAnimation.height.value = blockLayout.height;
-			collapseAnimation.opacity.value = withTiming(
-				1,
-				COLLAPSE_OPACITY_ANIMATION_CONFIG,
-				( completed ) => {
-					if ( completed ) {
-						collapseAnimation.height.value = withTiming(
-							BLOCK_COLLAPSED_HEIGHT,
-							COLLAPSE_HEIGHT_ANIMATION_CONFIG
-						);
-					}
-				}
-			);
-		}
 	};
 
-	const stopBlockDragging = () => {
-		collapseAnimation.height.value = withTiming(
-			collapseAnimation.initialHeight.value,
-			EXPAND_HEIGHT_ANIMATION_CONFIG,
-			( completed ) => {
-				if ( completed ) {
-					collapseAnimation.opacity.value = withTiming(
-						0,
-						COLLAPSE_OPACITY_ANIMATION_CONFIG
-					);
-				}
-			}
+	const stopDraggingBlock = () => {
+		draggingAnimation.opacity.value = withTiming(
```

Comment:
While testing on Android, I barely could see the opacity change. I tested in `DEV` mode but I wonder if we could add a delay here for both iOS and Android. I think the transition is smoother while the new block lands in its new position and the block gets selected.

Something like:

```suggestion
		draggingAnimation.opacity.value = withDelay(
			200,
			withTiming( 1, BLOCK_OPACITY_ANIMATION_CONFIG )
		);
```

What do you think?

Reply:
Good idea! When the blocks are moved the UI gets frozen for some milliseconds, which most likely is making less visible the fading animation. Adding a delay will help on ensuring that the animation is started when the blocks already landed 👍.

Reply:
I applied this change in https://github.com/WordPress/gutenberg/pull/40104/commits/b62fcb324ad32783e69ee0bbd3513fb2872e6248.